src/core/a-entity.js

```diff
@@ -525,6 +525,25 @@ var proto = Object.create(ANode.prototype, {
     }
   },
 
+  /**
+   * A mapping of attribute to handle component attributes and singular attributes
+   *
+   * @param {string} attr dot notation or singular 'color' or 'material.color'
+   * @returns {object|string|number} Value of type of component property.
+   */
+  getComputedAttributeFor: {
```

Comment:
Might be easier to keep this as a utility function within the animation module for now. (e.g., `getComputedAttributeFor(el, attribute)`.

Not sure if we want to fully support dot notation because if we do it here, then we have to do it for `setAttribute` as well to be consistent, and it'll increase API surface area / documentation / code to maintain. If we start to see a bigger need for dot notation, then we could re-eval.